src/lib/I18n.ts

```diff
@@ -0,0 +1,82 @@
+import { I18nManager } from 'react-native'
+
+import Countries from 'i18n-iso-countries'
+import { I18n as I18nJs } from 'i18n-js'
+import 'moment/min/locales'
+import { Scope, TranslateOptions } from 'i18n-js/typings/typing'
+import moment from 'moment'
+import * as RNLocalize from 'react-native-localize'
+
+import Log from './Log'
+
+Countries.registerLocale(require('i18n-iso-countries/langs/en.json'))
+Countries.registerLocale(require('i18n-iso-countries/langs/nl.json'))
+Countries.registerLocale(require('i18n-iso-countries/langs/de.json'))
+Countries.registerLocale(require('i18n-iso-countries/langs/es.json'))
+Countries.registerLocale(require('i18n-iso-countries/langs/fr.json'))
+
+interface Translations {
+  en: Object
+  nl: Object
+  de: Object
+  fr: Object
+  es: Object
+}
+
+interface Api {
+  setLanguage(language: string): void
+}
+
+/** Wrapper class around i18n-js, adding a few more methods */
+class I18n {
+  i18nJs = new I18nJs()
+
+  constructor(private translations: Translations, private api: Api) {
```

Comment:
In deze file heb ik de constructor aangepast en niet standaard een static instance geexport